### DISCUSSION TITLE
Animate single character with Text？

### DISCUSSION THREAD

**Discussion by bigxixi** [pmndrs/drei#1277] [Q&A]
<img width="421" alt="截屏2023-02-09 09 58 57" src="https://user-images.githubusercontent.com/22438240/217697698-d8ea925d-13fe-4b43-98cf-ac1e77fd18bf.png">

I want to do some text animations, basically the characters fly out from one single text line , Text3D is a little heavy for me, and the [Text](https://github.com/pmndrs/drei#text) looks good, but I can't access a single character in it.

Is there any way to do this?

**Comment by drcmda** [comment]
you would have to check troika-three-text, this is what text relies on.

btw what are all those cool looking gizmos?

**Reply by bigxixi** [reply]
Thanks for replying, I am reading troika-three-text docs.
The picture is a screen shot in Adobe After Effects, I use it to make animation videos.
